### PR TITLE
[REV-27] Make content an imagefield

### PR DESCRIPTION
**Pull request by andrevieira11**

### CODE REVIEW

**Review by g-esteves-rr — COMMENTED** [review]
I think you can only have one migration per PR, if you delete all new generated migration files, and run the command again, it will generate just one file with all the migrations.

File: photo/tests/test_serializers.py [hunk 1]
```diff
@@ -122,21 +122,23 @@ def test_contest_serialized_invalid_data(self):
...
-            "contest": submission.contest.id,
-            "content": submission.content,
-            "description": submission.description,
-        }
-
-        serializer = SubmissionSerializer(data=valid_serialized_data)
-
-        assert serializer.is_valid(raise_exception=True)
-        assert serializer.errors == {}
+    # @pytest.mark.django_db
```
Comment (g-esteves-rr) [inline photo/tests/test_serializers.py:125]: This is commented because you are waiting for the other ticket?

Comment (andrevieira11) [reply photo/tests/test_serializers.py:125]: This is still WIP, and this test is failling, I need to make some adjustments

**Review by g-esteves-rr — APPROVED** [review]
tested it and it was working fine

**Review by tiagocardoso98 — APPROVED** [review]
It is working. We need to start thinking about using git secret to add the .env to the repository.

**Review by kenvontucky — CHANGES_REQUESTED** [review]
already existing migrations shouldn't be altered or deleted. Please create a new migration.

**Comment by andrevieira11** [comment]
The code's migrations are fixed but there is a test missing - test_update_put - this test isn't working because the model has an imagefield isntead of a textfield. The idea behind the test is that we want to create a new imagefile in localstack, create a temporary file locally and send this local file on the PUT method, then compare with the retrieve object. For some reason the post says the "user", "contest" and "description" are missing

**Pull request closed without merging**